packages/jaeger-ui/src/utils/config/get-config.test.js

```diff
@@ -54,25 +55,31 @@ describe('getConfig()', () => {
     });
   });
 
-  describe('`window.getJaegerUiConfig` is a function', () => {
+  describe('index functions are injected by backend', () => {
     let embedded;
     let getJaegerUiConfig;
+    let capabilities;
+    let getJaegerStorageCapabilities;
```

Comment:
why do these need to be made global within the tests instead of being local vars inside beforeEach function? Only `embedded` is used in some tests.